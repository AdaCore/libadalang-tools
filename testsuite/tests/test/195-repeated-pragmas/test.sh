#!/usr/bin/env bash

set -e

STUB_LOCATION='obj/gnattest_stub/stubs/Input/input-stub_data.ads'

gnattest --stub -P ops.gpr 2> /dev/null
cat $STUB_LOCATION
gnattest --stub -P ops.gpr 2> /dev/null
cat $STUB_LOCATION
