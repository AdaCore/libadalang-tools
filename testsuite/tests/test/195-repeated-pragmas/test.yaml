description:
  gnattest test

driver: gnattest_driver
