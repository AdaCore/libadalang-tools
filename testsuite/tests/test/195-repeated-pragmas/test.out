--  begin read only
--  id:2.2/00/
--
--  This section can be used to add with clauses if necessary.
--
--  end read only

pragma Ada_2012;
--  begin read only
--  end read only

package Input.Stub_Data is

--  begin read only
   --  id:2.2/04/a5520b8be5540073/b568d47f2e244743/0/Read_Number/
--  end read only
   type Stub_Data_Type_Read_Number_a5520b_b568d4 is record
      Read_Number_Result : Integer;

      Stub_Counter : Natural := 0;
   end record;
   Stub_Data_Read_Number_a5520b_b568d4 : Stub_Data_Type_Read_Number_a5520b_b568d4;
   procedure Set_Stub_Read_Number_a5520b_b568d4
     (Read_Number_Result : Integer := Stub_Data_Read_Number_a5520b_b568d4.Read_Number_Result);
--  begin read only
--  end read only

end Input.Stub_Data;
--  begin read only
--  id:2.2/00/
--
--  This section can be used to add with clauses if necessary.
--
--  end read only

pragma Ada_2012;
--  begin read only
--  end read only

package Input.Stub_Data is

--  begin read only
   --  id:2.2/04/a5520b8be5540073/b568d47f2e244743/0/Read_Number/
--  end read only
   type Stub_Data_Type_Read_Number_a5520b_b568d4 is record
      Read_Number_Result : Integer;

      Stub_Counter : Natural := 0;
   end record;
   Stub_Data_Read_Number_a5520b_b568d4 : Stub_Data_Type_Read_Number_a5520b_b568d4;
   procedure Set_Stub_Read_Number_a5520b_b568d4
     (Read_Number_Result : Integer := Stub_Data_Read_Number_a5520b_b568d4.Read_Number_Result);
--  begin read only
--  end read only

end Input.Stub_Data;
